src/views/Graph.vue

```diff
@@ -101,7 +101,7 @@ const config = {
         'background-fit': 'contain contain',
         'background-image-opacity': function (node) {
           let opacity
-          node.data('running') > 0 ? opacity = 1 : opacity = 0.6
+          return node.data('running') > 0 ? 1.0 : 0.6
           return opacity
```

Comment:
Sorry I forgot to say earlier, but in which case, you can also remove the lines before and after.

Reply:
Oh, sure. I was just testing the GitHub "Commit this diff" feature. Quite cool :) Will edit the previous commit.